wasm2c/examples/rot13/Makefile

```diff
@@ -0,0 +1,9 @@
+# Use implicit rules for compiling C files.
+CFLAGS=-I../..
+rot13: main.o rot13.o ../../wasm-rt-impl.o
```

Comment:
I'm not very familiar with the wasm2c stuff.  Why is this called `wasm-rt-impl.o`?  Wouldn't `wasm-rt.o` be clear enough?

Reply:
My original thought was that `wasm-rt.h` just provides an interface, and could be implemented different ways. But I guess there's not much value in that.

Reply:
I see, but isn't object code by definition an implementation making this redundant?    If there were multiple implementations we might expect an adjective there such as wasm-rt-generic or wasm-rt-default maybe?  I might be missing something here.